### PR TITLE
Rotate-module added

### LINKED ISSUE #434: Rotate module
We should have a rotate module that accepts an angle. I'm sure there's a library we can use to make this easy. Maybe:

https://www.npmjs.com/package/imagejs

We'd love help with this one!

### PR DESCRIPTION
**Pull request by aashna27**
Fixes #434
Extremely sorry for being so late 😞

### CODE REVIEW

**Comment by jywarren** [comment]
No problem at all! Very interested to see this-- could you share a before/after screenshot of the module working? Thank you!

File: src/modules/Rotate/Module.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        var imagejs = require('imagejs');
+
+        function changePixel(r, g, b, a) {
+            return [r, g, b, a]
+        }
+
+        function extraManipulation(pixels) {
+            var rotate_value = (options.rotate)%360;
+
+            var bitmap = new imagejs.Bitmap({width: pixels.shape[0], height: pixels.shape[1]});
```
Comment (jywarren) [inline src/modules/Rotate/Module.js]: This'll be a nice example case for the other features of `imagejs`, which we could implement following this template!

**Comment by aashna27** [comment]
yes sure !!
before :
image
after: 
image

image

**Comment by jywarren** [comment]
this looks amazing. Do you think the rotation should be positive =
clockwise? Is there a strong convention we should follow or is this OK?
Thanks!

**Comment by aashna27** [comment]
I don't know of any convention, but yes the the module had it for anticlockwise.

File: src/modules/Rotate/info.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Rotate",
+    "description": "Rotates image by specified degrees",
+    "inputs": {
+      "rotate": {
+        "type": "integer",
+        "desc": "Angular value for rotation",
```
Comment (jywarren) [inline src/modules/Rotate/info.json]: Can you specify "in degrees" or "in radians" etc? Thank you!!! Then, i think this is ready for merge -- what do you think?

Comment (aashna27) [reply src/modules/Rotate/info.json]: yess sure ! I have updated 😄

**Comment by jywarren** [comment]
Just quickly rebasing this over the latest!

**Comment by jywarren** [comment]
Whew! Great work here! I'll push it up to https://sequencer.publiclab.org in a moment :-)

**Comment by jywarren** [comment]
Hooray!! https://sequencer.publiclab.org/examples/#steps=rotate{rotate:30}

**Pull request merged**